CHANGELOG.md

```diff
@@ -7,6 +7,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [UNRELEASED]
 
+### Docs
+
+- Synced with develop to ensure no files other than the ones related to CLI updates are modified
```

Comment:
This should not be in the changelog since it's not a change to the trunk

Reply:
removed it